community/merchants/index.md

```diff
@@ -85,7 +85,25 @@ meta_descr: merchants.descr
           </div>
         </div>
       </div>
-    </div>
+      <div class="right half no-pad-sm col-lg-6 col-md-6 col-sm-12 col-xs-12">
+        <div class="info-block">
+          <div class="center-xs">
+            <img class="merch" src="/img/merchants/basicswapdex.png" alt="BasicSwapDEX Logo" title="BasicSwapDEX">
+            <p>{% t merchants.basicswapdexdesc %}</p>
+          </div>
+          <div>
+            <p><b>{% t merchants.cardfoss %}</b> <span class="check"></span></p>
+            <p><b>{% t merchants.noncustodial %}</b> <span class="check"></span></p>
+            <p>{% t merchants.cardkyc %} <span class="check"></span></p>
+            <p><b>XMR &#8596; fiat:</b> <span class="cross"></span></p>
+            <p><b>{% t merchants.cardonion %}</b> <span class="cross"><b>*</b></span></p>
```

Comment:
```suggestion
            <p><b>{% t merchants.cardonion %}</b> <span class="cross"></span><b>*</b></p>
```

i broke it 🤫
maybe this works

or just drop the asterisk

Reply:
that works yes. notice how localmonero has, instead of an asterisk / check / cross: just  (Depends on seller). "Setup required" and call it a day? and eventually the same for Bisq?